app/static/pages/volunteer/VolunteerInfoDefinitions.js

```diff
@@ -111,5 +73,13 @@ oppiaFoundationWebsite.constant('VOLUNTEER_INFO', {
       'know that Oppia\'s engaging and equitable lessons can help solve this ' +
       'problem. Every volunteer matters at Oppia.',
     position: 'Coordinator, Lesson Creation'
+  }],
+
+  local_outreach: [{
```

Comment:
Do you think it is worth adding blurbs for Yamama and Sam / Rev Bankas? This PR doesn't need to block on that but it might be nice to solicit those and add them.

Reply:
Yes, I'd love to include them once we get quotes from them!